build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -1510,6 +1510,11 @@ message BatchUpdateBlobsRequest {
 
     // The raw binary data.
     bytes data = 2;
+
+    // The format of `data`. Must be `IDENTITY` (or unspecified), or one of the
+    // compressors advertised by the `CacheCapabilities.supported_compressor` if
```

Comment:
supported_compressor -> supported_batch_compressors , and I think inlined_compressed_blobs is no longer a field?

Reply:
Fixed.